src/Leaflet.Editable.js

```diff
@@ -795,7 +795,6 @@
 
         onFeatureAdd: function () {
             this.tools.editLayer.addLayer(this.editLayer);
-            if (this.feature.dragging) this.feature.dragging.enable();
```

Comment:
Seems a bit radical. It certainly breaks elsewhere. Have you run the tests?

I'd say at least we should check `this.feature.option.draggable` value.

Reply:
Yea, that would probably be safer although I don't unterstand why dragging needs to be handled by Leaflet.Editable because it is the key feature of Leaflet.Path.Drag.

All enableDragging tests fail, as expected, because there's no `draggable: true` in the path options of the tests. Do you need another PR for this?

Reply:
> I don't unterstand why dragging needs to be handled by Leaflet.Editable because it is the key feature of Leaflet.Path.Drag.

The tricky part is that one may want a path to be draggable when editable, but not by default.
So I think I'm wrong suggestion to check `options.draggable` which define the behaviour of the path when not editable.
I think it's fair that the default behaviour of Leaflet.Editable is to enable dragging in the same time editing is enabled, because from the Leaflet.Editable point of view dragging is part of editing (*if* Path.Drag is around of course).
I need a bit more thinking.

Reply:
How is your thinking going? :)

Reply:
Just to be sure of you user case: you have Leaflet.Path.Drag installed, but you don't want the paths to be draggable while in edit mode, correct?

Reply:
Yea. I want the paths to be draggable as soon I want them to be draggable :-) The main problem in my use case is that people on mobile accidentally drag the path while they wanted to drag the vertices. I've got a lot of complaints about this in the last 12 months.

Reply:
OK, I see.

I'm balanced about this being a bug or a feature.
Dragging a path is part of the edit workflow, as seen by Leaflet.Editable.
Maybe making this automagic when `dragging` is discovered was a bad design decision. But I feel like wanting to have drag enabled BUT not while in edit mode is an edge case.
Anyway, I think this should still be the default as for now.
One possibility is to add an option that controls that (something like `allowDragEdit`), but usually adding a "behaviour" option is the sign of a bad design somewhere else.
Leaflet.Editable is basically "hookable" in a lot of ways possible, which is its main design principle.
So maybe here this is what should be used, either by hooking the `editable:enable` event, or by subclassing `L.PolygonEditor` and friends, to override the `onFeatureAdd` behaviour.

In other summarized words, I think discussing the decision to automagically enable dragging while editing is fair, but we should either keep this behaviour as is or remove it; but I feel like adding more option to handle such behaviour will end in a fucked up API quite soon.
So, short term, how do you feel about either subclassing or using events to deal with that without touching the core?

Reply:
Thanks. As you suggested, I subclassed `L.Editable.BaseEditor` :+1: